script/testing/junit/traces/cte.sql

```diff
@@ -0,0 +1,36 @@
+WITH cte(x) AS (SELECT 1) SELECT x FROM cte;
```

Comment:
What's the source on these? Are some sqlite test cases and some our own?

Reply:
These are all our own tests, tested against postgres